### ISSUE TITLE
Code Review

### ISSUE THREAD

**Issue by WenjieLuo2333** [294486709/Youtube_Analyzer#2]
1.why input size is 100\*19? （In your code is 100*18, but the nn_network requires it to be 100\*19,so i change it into 100\*19.) So you assume each video title has at most 19 words?<br/>
2.It takes me a lot of time to understand your code because i cant find a clear build instruction and readme.<br/>
3.I think the xtest, ytest in your ```pure_NN.py``` file should be validation sets but not test sets,right?Because you run test on xtest/ytest every iteration. 
### Can you provide a test set and a dependent module to load the model and test on test sets?Because the accuracy of validation sets sometimes can be much higher than test accuracy.

4.Depend my own understanding of your model, each title is convert into a 1\*1900 vector, most of the value may be useless.(Not all title has 19 words).And the NN layer you use contains a [input_size,1000] filter. I think it's similar to put a [19\*100] image into a Conv2D with 1000 19\*100 filters. And do it with more than 10 layers. Have you considered the filter size might be too large? The network might suffer from over-fiting?(i have no test set so i cannot make sure whether over-fit or not)
### I have a proposal maybe you can try<br/> 
keep the word vector in 19\*100 format and see it similar to picture. Then use Conv2D(filter size should be k*100,k<19) to extract the feature map(one layer is enough). And then apply pooling(enlarge the filters‘ receptive field) and dropout(prevent overfiting) to it. Finally get result by a fclayer.
you can also leave the word vector in forms of 1\*1900.(build in embedding layer is ok). Then use conv1d layers to extract features.
